SkillOnCall/booking/views.py

```diff
@@ -68,3 +68,47 @@ def view_bookings(request):
     bookings = Booking.objects.filter(customer_id=customer)
 
     return render(request, 'booking/all_bookings.html', {'bookings': bookings})
+
+@login_required
+def my_allocation(request):
+    provider = request.user.customer.serviceprovider
+    bookings = Booking.objects.filter(service_provider_id=provider)
+
+    return render(request, 'booking/my_allocation.html', {'bookings': bookings})
+
+@login_required	
+def confirm_booking(request, booking_id):
```

Comment:
```suggestion
@login_required	
def confirm_booking(request, booking_id):
```

Hello @shubhpatel232 ,

I have an idea when provider cancal or confirm the booking we don't user the login_require decorator ?

What do think let me know so i can make changes in this code and try to make that possible

Reply:
to allow this routes without login_require we need to introduce something like access token instead of direct using ID. so yes you can improve this in new PR.

Reply:
@shubhpatel232  okay its good idea to improve this in new PR. so i'll merge this PR